util/opentelemetry-util-http/src/opentelemetry/util/http/__init__.py

```diff
@@ -15,9 +15,12 @@
 from os import environ
 from re import compile as re_compile
 from re import search
-from typing import Iterable
+from typing import Iterable, List
 from urllib.parse import urlparse, urlunparse
 
+OTEL_CAPTURE_REQUEST_HEADERS = "OTEL_CAPTURE_REQUEST_HEADERS"
```

Comment:
Do we want to further split this to client/server like Java does?

Reply:
I think it is good to split as this will make the feature consistent between different sdks. Also, will keep the env vars same as java sdk.

Not sure but, when I first read the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#http-request-and-response-headers) it felt like its only talking about the server side. Please share your thoughts :)

Reply:
Seems like we are going with the general approach for being consistent with other language sdks like [here](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/925#discussion_r814056927) so it would make sense to follow suit.